### ISSUE TITLE
Non-interactive command line is truncated at 260 characters

### ISSUE THREAD

**Issue by jakejustus** [PowerShell/Win32-OpenSSH#161]
Using 2_25_2016 release
Client : Win64-OpenSSH ssh.exe executable on Windows 10
Server : Win64-OpenSSH sshd.exe executable on Windows 2012R2
May be related to issue #115 

Executing this command line :

```
ssh.exe user@host "cmd /c echo       20        30        40        50        60        70        80        90       100       110       120       130       140       150       160       170       180       190       200       210       220       230       240       250       260"
```

Generates this output :

```
      20        30        40        50        60        70        80        90       100       110       120       130       140       150       160       170       180       190       200       210       220       230       240       250       260????????????4???
```

Removing the trailing character generates a correct output, so it seems the ssh.exe executable doesn't accept more than 260 characters. Is that right ? Or does it come from my shell ?

**Comment by hach-que** [comment]
I'm running into this as well.  It's a major issue for us as we're trying to use SSHD to control a Windows build agent, and some of our build commands are longer than this length.

**Comment by rgl** [comment]
this now seems to work with v0.0.5.0, can you confirm?

**Comment by gquerret** [comment]
Issue seems to be fixed in 0.0.5.0, I was able to execute a long command line in a small test environment.
